### PR TITLE
Updates to latest zipkin that defaults to Lens UI

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Here's a birthday present on behalf of zipkin
https://github.com/openzipkin/openzipkin.github.io/wiki/OpenZipkin's-5-year-anniversary

I don't have an account, but I tried with dummy credentials and seems to
wire up OK.

```bash
STORAGE_TYPE=logzio \
LOGZIO_ACCOUNT_TOKEN=icecream \
LOGZIO_API_TOKEN=candy \
java -Dloader.path='logzio.jar,logzio.jar!lib' -Dspring.profiles.active=logzio -cp zipkin.jar org.springframework.boot.loader.PropertiesLauncher
```

Note: I fixed the toString, but the health check should really be implemented at some point
```bash
$ curl -s localhost:9411/health
{
  "status" : "UP",
  "zipkin" : {
    "status" : "UP",
    "details" : {
      "LogzioStorage{logzioApiHost=https://api.logz.io/v1/search}" : {
        "status" : "UP"
      }
    }
  }
}
```

Screenshot 2020-04-19 at 12 11 29 PM

```
2020-04-19 12:11:06.388  WARN 4542 --- [cking-tasks-1-2] z.s.i.BodyIsExceptionMessage             : Unexpected error handling request.

java.lang.IllegalStateException: response for aggregation failed: {"code":401,"message":"HTTP 401 Unauthorized","requestId":"4c2cdb39-89c0-4132-8530-32bf2b753f45"}
	at zipkin2.storage.logzio.client.HttpCall.parseResponse(HttpCall.java:131) ~[zipkin-storage-logzio-0.0.5.jar!/:0.0.5]
	at zipkin2.storage.logzio.client.HttpCall.doExecute(HttpCall.java:70) ~[zipkin-storage-logzio-0.0.5.jar!/:0.0.5]
	at zipkin2.Call$Base.execute(Call.java:380) ~[zipkin-2.21.1.jar!/:?]
	at zipkin2.Call$FlatMapping.doExecute(Call.java:276) ~[zipkin-2.21.1.jar!/:?]
	at zipkin2.Call$Base.execute(Call.java:380) ~[zipkin-2.21.1.jar!/:?]
	at zipkin2.Call$Mapping.doExecute(Call.java:237) ~[zipkin-2.21.1.jar!/:?]
	at zipkin2.Call$Base.execute(Call.java:380) ~[zipkin-2.21.1.jar!/:?]
	at zipkin2.Call$Mapping.doExecute(Call.java:237) ~[zipkin-2.21.1.jar!/:?]
	at zipkin2.Call$Base.execute(Call.java:380) ~[zipkin-2.21.1.jar!/:?]
	at zipkin2.server.internal.ZipkinQueryApiV2.getTraces(ZipkinQueryApiV2.java:147) ~[classes!/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at com.linecorp.armeria.internal.server.annotation.AnnotatedService.invoke(AnnotatedService.java:294) ~[armeria-0.99.3.jar!/:?]
	at com.linecorp.armeria.internal.server.annotation.AnnotatedService.lambda$serve0$8(AnnotatedService.java:276) ~[armeria-0.99.3.jar!/:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478) ~[?:?]
	at com.linecorp.armeria.common.RequestContext.lambda$makeContextAware$3(RequestContext.java:379) ~[armeria-0.99.3.jar!/:?]
	at com.linecorp.armeria.internal.common.metric.TimedRunnable.run(TimedRunnable.java:63) [armeria-0.99.3.jar!/:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.48.Final.jar!/:4.1.48.Final]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

### CODE REVIEW

File: module/storage-logzio/src/main/resources/zipkin-server-logzio.yml [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+spring:
+  main:
+    # ZipkinLogzioStorageModule overrides the bean named 'storage'
+    allow-bean-definition-overriding: true
 zipkin:
+  internal:
+    module:
+      logzio: zipkin.module.storage.logzio.ZipkinLogzioStorageModule
   storage:
-    strict-trace-id: ${STRICT_TRACE_ID:true}
```
Comment (adriancole) [inline module/storage-logzio/src/main/resources/zipkin-server-logzio.yml:4]: this was redundant to the main file here https://github.com/openzipkin/zipkin/blob/master/zipkin-server/src/main/resources/zipkin-server-shared.yml#L91

File: pom.xml [hunk 2]
```diff
@@ -12,12 +12,12 @@
     <properties>
         <zipkin-logzio-version>0.0.5</zipkin-logzio-version>
         <!-- make sure this matches zipkin's version -->
-        <zipkin.version>2.16.0</zipkin.version>
```
Comment (adriancole) [inline pom.xml:15]: lens was available last august when this version was released, but it was optional (you'd have to click Try Lens) the current version it is default

File: storage-logzio/src/test/java/zipkin2/storage/logzio/LogzioSpanStoreTest.java [hunk 3]
```diff
@@ -1,49 +1,38 @@
...
+    @Rule public MockWebServer mockWebServer = new MockWebServer();
 
     private static String apiToken = "not-a-real-api-token";
     private static LogzioStorageParams params = new LogzioStorageParams();
     private static LogzioStorage storage;
     private static LogzioSpanStore spanStore;
 
-    @BeforeClass
-    public static void setup() {
-        logger.info(LogzioStorage.ZIPKIN_LOGZIO_STORAGE_MSG + "Setting up test environment");
```
Comment (adriancole) [inline storage-logzio/src/test/java/zipkin2/storage/logzio/LogzioSpanStoreTest.java:33]: by default, the junit rule already logs status messages. there's no need to do that again here, knowing this.

**Comment by barakm** [comment]
Outdated. Closing.

**Pull request closed without merging**